ocaml/xapi/storage_impl.ml

```diff
@@ -187,20 +163,18 @@ module Vdi = struct
       Vdi_automaton.Detached
 
   (** [get_dp_vm] returns [vm] of type Vm.t. If the mapping between a datapath and vm exists, it will return the vm. Owtherwise a blank vm will be returned*)
-
   let get_dp_vm dp t =
     let _vm = List.assoc_opt dp t.dpv in
     match _vm with
     | Some vm ->
         vm
     | None ->
-        debug
-          "[Vdi.get_dp_vm] No vm was found on the datapath please restart host \
-           to fix" ;
+        warn
+          "[Vdi.get_dp_vm] No vm was found associated with a datapath. Restart \
+           the host if this is a recent upgrade" ;
```

Comment:
```suggestion
          "%s: No vm was found associated with a datapath. Has xapi been restarted \
            since it was upgraded?" __FUNCTION__;
```

Reply:
Done